[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\selke\\AppData\\Local\\Temp\\quarto-session4404fac3675f368e'\n"]
